doc/decisions/frontend_architecture_pattern.md

```diff
@@ -0,0 +1,34 @@
+# Frontend Architecture Pattern
+
+## Problem
+
+Permaplant needs to be designed with a consistent and maintainable architecture pattern.
+
+## Constraints
+
+- The architecture pattern should be easy to understand and follow
+- The architecture pattern should be easy to test
+- The architecture pattern should be easy to scale
+- The architecture pattern should be suitable for a large, complex application
+- The architecture pattern should promote code reuse
+- The architecture pattern should be well-documented and widely used in the React community
+
+## Considered Alternatives
+
+- [Flux](https://reactjs.org/blog/2014/05/06/flux.html)
```

Comment:
Please write a sentence why we won't use it.

Reply:
@buenaflor Will you look into this?

Reply:
@absurd-turtle 
May I give an answer to that.
Flux is what was used before Redux got invented. It is pretty similar. Redux implements the Flux concepts in a library with a little more constraints.
The same reasons apply here, why we should not use Flux.

Reply:
Please simply push such statements into the PR. Later readers will not read what we wrote here but only what is in the decision text.